lib/sqlalchemy/orm/strategy_options.py

```diff
@@ -2319,7 +2319,9 @@ def contains_eager(*keys: _AttrType, **kw: Any) -> _AbstractLoad:
 
 
 @loader_unbound_fn
-def load_only(*attrs: _AttrType, raiseload: bool = False) -> _AbstractLoad:
+def load_only(
+    *attrs: QueryableAttribute[Any], raiseload: bool = False
```

Comment:
revert all these changes, and simply replace "str" from the `_AttrType` alias with `Literal['*']`

Reply:
I didn't want to touch the other cases where `_AttrType` is being used, so that's why I did this. If it's safe to change this in the alias, that's obviously the better solution. I'll update it accordingly.

Reply:
well we should have and/or add tests for all elements that use `_AttrType`, then there won't be a problem

Reply:
typing tests are plain python files in test/typing/plain_files/ .  if there is not already a file that is focused on loader options like `defer()`, `joinedload()`, etc. then one should be added